src/internal_events/pulsar.rs

```diff
@@ -7,6 +8,10 @@ pub struct PulsarEncodeEventFailed<'a> {
 
 impl<'a> InternalEvent for PulsarEncodeEventFailed<'a> {
     fn emit_logs(&self) {
-        debug!(message = "Event encode failed.", error = ?self.error);
+        debug!(message = "Event encode failed.", error = %self.error);
```

Comment:
I think this should be `warn!`, and do we need `dropping event` too?

Reply:
Good catch, thanks. Changed to `error!`, added `dropping event`.